tests/test_charm.py

```diff
@@ -240,19 +250,17 @@ def test_charm_setup_gitlab_application(
             self.harness.charm.unit.status.message, "failed to create application on gitlab"
         )
 
-        # `ssl.get_server_certificate` on GitLab failing:
+        # `_mock_ssl.getpeercert` on GitLab failing:
```

Comment:
it's not that we test that the mock_ssl.getpeercert is failing, but rather simulate the fact that the actual ssl.getpeercert is. So, the comment should be ``sslsock.getpeercert on GitLab failing:``